<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoFormDialog</class>
 <widget class="QWidget" name="InfoFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>1600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>1920</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>1920</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>1600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralWidget {
border-image: url(:/BDPassPrinter/Resources/inputFormBackGround.jpg);
}
</string>
   </property>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>676</y>
      <width>470</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>32</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneEdit">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>872</y>
      <width>470</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>32</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>676</y>
      <width>228</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/BDPassPrinter/Resources/step_button.png);
color: white;
border-radius:10px</string>
    </property>
    <property name="text">
     <string>姓 名</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>872</y>
      <width>228</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/BDPassPrinter/Resources/step_button.png);
color: rgb(255, 255, 255);
border-radius:10px</string>
    </property>
    <property name="text">
     <string>电 话</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>1100</y>
      <width>228</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/BDPassPrinter/Resources/step_button.png);
color: rgb(255, 255, 255);
border-radius:10px</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BDPassPrinter.qrc"/>
 </resources>
 <connections/>
</ui>
